NotifyTests/UnitTests/NotificationUnitClientTests.cs

```diff
@@ -448,5 +448,46 @@ void mockRequest(String content)
                     };
                 }));
         }
+
+		[Test, Category("Unit/NotificationClient")]
+		public void SendEmailNotificationWithReplyToIdGeneratesExpectedRequest()
+		{
+			Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
+				{
+					{ "name", "someone" }
+				};
+			JObject expected = new JObject
+			{
+				{ "email_address", Constants.fakeEmail },
+				{ "template_id", Constants.fakeTemplateId },
+				{ "personalisation", JObject.FromObject(personalisation) },
+				{ "reference", Constants.fakeNotificationReference },
+                { "email_reply_to_id", Constants.fakeReplyToId}
+			};
+
+			mockRequest(Constants.fakeTemplatePreviewResponseJson,
```

Comment:
What does `Constants.fakeTemplatePreviewResponseJson` do here, is it needed? I've also seen it in other tests so might have to update those if they are not using it properly